lib/structures/Guild.js

```diff
@@ -899,34 +899,41 @@ class Guild extends Base {
 
     toJSON(props = []) {
         return super.toJSON([
+            "name",
+            "verificationLevel",
+            "region",
+            "icon",
             "afkChannelID",
             "afkTimeout",
-            "banner",
-            "channels",
             "defaultNotifications",
-            "description",
-            "emojis",
-            "explicitContentFilter",
-            "features",
-            "icon",
+            "mfaLevel",
             "joinedAt",
+            "ownerID",
+            "systemChannelID",
+            "splash",
+            "banner",
+            "unavailable",
             "large",
-            "maxMembers",
             "maxPresences",
-            "memberCount",
+            "channels",
             "members",
-            "mfaLevel",
-            "name",
-            "ownerID",
-            "preferredLocale",
-            "premiumSubscriptionCount",
-            "premiumTier",
-            "region",
+            "memberCount",
             "roles",
-            "splash",
-            "unavailable",
-            "vanityURL",
-            "verificationLevel",
+            "features",
+            "emojis",
+            "explicitContentFilter",
+            "premiumTier",
+            "premiumSubscriptionCount",
+            "preferredLocale",
+            "description",
+            "maxMembers",
+            "publicUpdatesChannelID",
+            "rulesChannelID",
+            "maxVideoChannelUsers",
+            "widgetEnabled",
+            "widgetChannelID",
+            "approximateMemberCount",
+            "approximatePresenceCount",
```

Comment:
Keep this alphabetical

Reply:
Also you're missing `shard`, `voiceStates`, `applicationID`, `autoRemoved`, `emojiCount`, `primaryCategoryID`, `primaryCategory`, `categories`, `keywords`, `pendingVoiceStates`, `discoverySplash`, `systemChannelFlags`, `vanityURL`